### DISCUSSION TITLE
"Feels Like" not showing up as an option for secondary info attribute in HA Weather Forecast Card

### DISCUSSION THREAD

**Discussion by francis110** [jaydeethree/Home-Assistant-weatherdotcom#26] [Ideas]
I was able to make this an option by adding this to weather.py

`    @property
    def native_apparent_temperature(self) -> float:
        """Return the feelslike in native units."""
        return self.coordinator.get_current(FIELD_FEELSLIKE) `

However, only the value is displayed. When looking at the entity it appears as (degrees) F, but this is not coming through in the card.

I think this is a problem with HA, but I don't know enough to really say. Just a tinkerer.

![Screenshot 2024-02-16 164516](https://github.com/jaydeethree/Home-Assistant-weatherdotcom/assets/23224094/c802374f-dd28-434b-b065-1c320e355678)

**Comment by jaydeethree** [comment]
I have no control over how this card displays "feels like" (or any other) data - you would need to open a bug with the card's maintainers. That said, you are correct that this integration is not setting the `native_apparent_temperature` property even though it has that data, so that's something that I will fix when I have some time.

**Reply by jaydeethree** [reply]
Update: version 1.1.5 is now available and includes the `native_apparent_temperature` property (as well as a few other properties that were missing).

**Reply by francis110** [reply]
Thank you! sorry about the post in issues. I was thinking this was more of a request and must have accidently posted it when I was copying it an moving it here!
